gulpfile.js

```diff
@@ -20,23 +20,11 @@ var fs = require('fs');
 var yargs = require('yargs');
```

Comment:
Please add some documentation to this file explaining to devs what it's meant to be used for and what it's not, otherwise no one will know what to put where.

Reply:
@vojtechjelinek -- flagging this comment for a (longish?) docstring at the top of the file, in case you missed it before.

Reply:
Done.